pkg/controllers/management/kontainerdrivermetadata/data.go

```diff
@@ -328,6 +360,85 @@ func (md *MetadataController) createOrUpdateAddonCRD(addonName, template string,
 	return nil
 }
 
+func (md *MetadataController) createOrUpdateCisConfigCRD(
```

Comment:
nit: CRD should not be in the name of the func

Reply:
I think it's easier to read and understand the code with CRD in place, let's leave it this way :)